src/database/migration/v2.js

```diff
@@ -188,6 +132,61 @@ const dataMigrations = [
       });
     },
   },
+  {
+    version: '2.3.6',
+    migrate: database => {
+      // Find all the transactions with no item batches, if any
+      const transactionBatches = database
+        .objects('TransactionBatch')
+        .filtered('itemBatch == $0', null)
+        .snapshot();
+      // Do nothing if there aren't any.
+      if (!(transactionBatches.length > 0)) return;
```

Comment:
not a biggy, but this line is not required

Reply:
d4989fd806f9a91a61716c13b8d3352c76b0927d